mgmtsystem/hawkular.py

```diff
@@ -52,13 +56,15 @@ class Hawkular(MgmtSystemAPIBase):
     }
 
     def __init__(self,
-            hostname, protocol="http", port=8080, entry="hawkular/inventory", **kwargs):
+            hostname, protocol="http", port=8080, inventory_entry="hawkular/inventory",
+            alerts_entry="hawkular/alerts", **kwargs):
```

Comment:
@hhovsepy I guess we no need to pass `inventory_entry`, `alerts_entry`, because this is static REST URI will never change. We have other sub modules also.